source/common/common/utility.h

```diff
@@ -221,6 +221,15 @@ class StringUtil {
    */
   static absl::string_view trim(absl::string_view source);
 
+  /**
+   * Removes any specific trailing characters from the end of a string.
```

Comment:
nit: ... from the end of a string view

(since it doesn't modify the underlying string)